docs/source/api/pystiche_papers.ulyanov_et_al_2016.rst

```diff
@@ -1,6 +1,10 @@
 ``pystiche_papers.ulyanov_et_al_2016``
 =========================================
 
+.. _table-branches-ulyanov_et_al_2016:
+
+.. _table-hyperparameters-ulyanov_et_al_2016:
+
```

Comment:
This should be resolved when #157 lands. Thus, you should remove this and simply rebase when its ready.